t/utils/utils.go

```diff
@@ -336,10 +337,36 @@ func ImportTestAccount(keystoreDir, accountFile string) error {
 	}
 
 	dst := filepath.Join(keystoreDir, accountFile)
-	err := ioutil.WriteFile(dst, static.MustAsset("keys/"+accountFile), 0644)
+	err := copyFile(dst, path.Join(params.GetStatusHome(), "static/keys/", accountFile))
 	if err != nil {
 		logger.Warn("cannot copy test account PK", "error", err)
 	}
 
 	return err
 }
+
+func gettestConfigFromFile(fileName string, config *testConfig) error {
```

Comment:
`getTestConfigFromFile`?

Reply:
I idea was to use the variable name `testConfig` as is, but sure I can change to this.